CMakeLists.txt

```
@@ -46,6 +46,7 @@ include(CTest)
 
 # Options
 option(HOLOHUB_DOWNLOAD_DATASETS "Download datasets" ON)
+option(HOLOHUB_BUILD_CPP "Build support for C++" ON)
```

Nitpick, from internal discussion C++ operators needed by an app will not be impacted

```suggestion
option(HOLOHUB_BUILD_CPP "Application build support for C++" ON)
```

---

Actually, this is not true. If anyone wants to use for operators, they can. It's just that currently it mostly makes sense for our applications. For instance, if they have python version of an operators that is implemented independently of C++ operator (not bindings), they should be able to use this.